### PR TITLE
Stick connected video rooms to the top of the room list

### PR DESCRIPTION
**Pull request by robintown**
Thanks to Travis for helping me avoid the dragons :dragon:

Type: enhancement

---
Here's what your changelog entry will look like:

## ✨ Features
 * Stick connected video rooms to the top of the room list ([\#8353](https://github.com/matrix-org/matrix-react-sdk/pull/8353)).

Preview: https://pr8353--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Comment by kerryarchibald** [comment]
Could this reasonably be split into a few smaller PRs?

**Comment by robintown** [comment]
The only actual change here is https://github.com/matrix-org/matrix-react-sdk/commit/783ba837f65256e877a8cc718bc94cdb40993756, which itself is about 150 lines changed. The rest of the commits are just https://github.com/matrix-org/matrix-react-sdk/pull/8287, which I've included here in order to not generate additional conflicts. I would of course want to wait for https://github.com/matrix-org/matrix-react-sdk/pull/8287 to land before merging this.

File: res/css/views/voip/_VideoLobby.scss [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+                        mask-size: $size;
+                        mask-position: center;
+                        background-color: $video-lobby-system;
+                        height: 100%;
+                        width: 100%;
+                    }
+                }
+
+                &.mx_VideoLobby_deviceButtonWrapper_active {
+                    .mx_VideoLobby_deviceButton, .mx_VideoLobby_deviceListButton {
```
Comment (luixxiul) [inline res/css/views/voip/_VideoLobby.scss:147]: Should this not be broken into two lines according to https://github.com/vector-im/element-meta/blob/travis/temp-code-style-web/web-code-style.md#scss (no.6) ?

**Review by kerryarchibald — APPROVED** [review]
LGTM

**Pull request merged**